examples/simple-app/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "simple-app",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "parcel serve --no-cache index.html"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "virtuoso-design-system": "^1.0.2-canary.c3286ae.0"
```

Comment:
Should we do the broadest possible version (e.g. `^1`) for forward compatibility?

Reply:
As soon as we get the latest release out ;)